test/cbmc/run_proofs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+coreJSONDir="coreJSON"
+
+UNWIND_COUNT=${UNWIND_COUNT:-10}
+
+#If coreJSON not found, clone it
+if [ ! -d "$coreJSONDir" ]; then
+    git clone https://github.com/FreeRTOS/coreJSON.git
```

Comment:
This clone should be changed to perform a [shallow clone](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt), and it should also perform a [sparse clone](https://git-scm.com/docs/git-sparse-checkout#Documentation/git-sparse-checkout.txt-codegitsparse-checkoutsetMYDIR1SUBDIR2code) of just the necessary source files from coreJSON

Reply:
It should also be cloning the tagged version of the repository, likely the LTS release version